L1Trigger/TrackFindingTracklet/src/PurgeDuplicate.cc

```diff
@@ -681,3 +744,46 @@ std::vector<const Stub*> PurgeDuplicate::getInventedSeedingStub(unsigned int iSe
   }
   return newStubList;
 }
+
+// Tells us the variable bin to which a track would belong
```

Comment:
Why do we need both functions findVarRInvBin() and findOverlapRInvBins()? Isn't it enough to have one function returning a vector of the (overlapping) bins a given Tracklet is in?

Reply:
findVarRInvBin() is used in the case that if we have a track that would be sent to output for 2 different bins, then it's only sent to output if the varrinvbin it's in matches the current bin in the loop. I can't think of another way to incorporate this into findOverlapRInvBins(), but any suggestions would be welcome.